### PR TITLE
[Merged by Bors] - prepass section

### PR DESCRIPTION
**Pull request by IceSentry**
This currently takes a lot of vertical space. Personally I like it, but I'm a bit biased 😉 

image

I reused the authors css for the images subtext, there's probably a better way to do this.

### CODE REVIEW

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 1]
```diff
@@ -416,6 +416,23 @@ The Bevy ECS team has worked closely with `@jakobhellerman`, the author of [`bev
 
 It's a great tool that we are looking to build on to create a first party solution: you should strongly consider adding it to your toolbox.
 
+## Depth and normal prepass
+
+<div class="release-feature-authors">authors: @icesentry, @superdump, @robtfm, @JMS55</div>
+
+<video controls loop><source  src="force_field.mp4" type="video/mp4"/></video>
+<p class="release-feature-authors">This effect uses the depth from the prepass to find the interesection between the ground and the force field</p>
```
Comment (LiamGallagher737) [inline content/news/2023-03-04-bevy-0.10/index.md:424]:
```suggestion
<p class="release-feature-authors">This effect uses the depth from the prepass to find the intersection between the ground and the force field</p>
```

**Review by LiamGallagher737 — COMMENTED** [review]
I don't think you should be worried about the use of vertical space, the images and video make it much easier for someone with limited rendering knowledge (like me) to understand.

**Comment by james7132** [comment]
Just wanted to drop in and say this is some really nice eye candy, great presentation of the feature!

**Comment by cart** [comment]
I'm happy with the current state of things. @IceSentry mentioned some tweaks, but you can follow up if you find the time!

**Pull request closed without merging**